src/content/en/updates/2017/12/modulepreload.md

```diff
@@ -0,0 +1,138 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>Loader #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time. This article looks at how it works and why it's better for modules than &lt;link rel="preload"&gt;. #}
+
+# Preloading modules {: .page-title }
+
+{% include "web/_shared/contributors/sgomes.html" %}
+
+Browsers are finally starting to natively support ES2015 modules, both with static and
+dynamic import support. This means it's now possible to write module-based
+JavaScript that runs natively in the browser, without transpilers or bundlers.
+
+Module-based development offers some real advantages in terms of cacheability,
+helping you reduce the number of bytes you need to ship to your users with a new release.
+The finer granularity of the code also helps with the loading story, by letting you
+prioritise the critical code in your application.
+
+That said, dependency trees can get deep very quickly, and there wasn't really a
+good way of declaratively preloading modules until now.
+Chrome 64 ships with `<link rel="modulepreload">` enabled by default, a module-specific
+version of `<link rel="preload">` that solves a number of the latter's problems.
+
+Warning: It's still very much early days for modules in the browser, so while we
+encourage experimentation, we definitely do not recommend using this technology in
+production just yet!
+
+## Wait, what's `<link rel="preload">`?
+
+Chrome added support for `<link rel="preload">` back in version 50, as a way of
```

Comment:
link to https://www.chromestatus.com/features/5757468554559488

Reply:
Done.